src/TidierDB.jl

```diff
@@ -413,7 +414,7 @@ end
 function connect(::duckdb, token::String)
     if token == "md:" 
         return DBInterface.connect(DuckDB.DB, "md:")
-    elseif endswith(token, ".duckdb")
+    elseif endswith(token, ".duckdb") || endswith(token, ".duck.db")
```

Comment:
imho, `connect` should take the protocol as part of token, i.e.

```
connect(duckdb(), "duckdb://path/to/test.db")
connect(duckdb(), "md://...")
```

Currently, if the token doesn't match the `if` or `elseif` it opens a motherduck auth page which is a confusing user experience (imo).

Reply:
I agree it's confusing. I might move md tokens to a separate function w a slightly different call to make it less likely this happens. Sorry for the confusion

Reply:
this should now be fixed 
```
connect(duckdb(), "./my_database.duckdb")
connect(duckdb(), "./my_database.duck.db")
connect(duckdb(), "md:") # established
connect(duckdb(), "md://...") # new
```

I have also added docs around it in the getting started page (will populate once merged). thx again